src/www/ui/admin-obligation-file.php

```diff
@@ -262,6 +262,8 @@ function ObligationTopic($topic)
       $ob .= "<td align=center>$associatedLicenses</td>";
       $ob .= "<td align=center>$candidateLicenses</td>";
       $ob .= "</tr>";
+      $ob_type = $row['ob_type'];
+      $ob_classification = $row['ob_classification'];
```

Comment:
Can you please describe the use of these two new variables? They seems to be unused.

Reply:
The variables are used in src/www/ui/template/admin_obligation-upload_form.html.twig. Line 28 and 45.
These variables are the actual Obligation Type (ob_type) and Obligation Classification (ob_classification) related to that particular obligation fetched from the database

Reply:
Earlier these two parameters were hardcoded as string values on line 28 and 45 as 'Obligation' and 'green' respectively

Reply:
The values in the `*.twig` templates goes in the form of array keys. If you want to pass some value to the twig file, you should add it to any one of the array near https://github.com/fossology/fossology/blob/master/src/www/ui/admin-obligation-file.php#L365 as it is done in https://github.com/fossology/fossology/blob/master/src/www/ui/admin-obligation-file.php#L338.

Reply:
Hello @hastagAB, are you planning to working on this?

Reply:
The selectsingle twig function on line 28 and 45 in the `admin_obligation-upload_form.html.twig` file is expecting a selected value associated with that particular Obligation. But in the current version of the code, the function is being passed a static string value. 
In line 28 it is currently 'Obligation' so no matter what the actual value is the dropdown will always show 'Obligation'
In line 45 it is currently 'green' so no matter what the actual value is in the database the dropdown will always show 'green'

I have replaced these two static strings with the corresponding variables defined in `admin-obligation-file.php` which stores the actual value associated with the particular Obligation.

Reply:
I did understood your point, what I was trying to convey is that the twig needs input for placeholders as an index in an array before rendering the twig template. You can check the code lines I've share before.